<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegrecionLinea</class>
 <widget class="QMainWindow" name="RegrecionLinea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegrecionLinea</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>382</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>ecuacion</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Borrar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Salir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Vivaldi</family>
           <pointsize>14</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Vivaldi</family>
           <pointsize>14</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_11"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_12"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_13"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_14"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>RegrecionLinea</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
